File: app/src/main/java/com/example/a18/path/wave/WaveActivity.java
package com.example.a18.path.wave;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.example.a18.path.R;

public class WaveActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_bezier);
        final WaveView viewById = findViewById(R.id.bezier);
        viewById.post(new Runnable() {
            @Override
            public void run() {
                viewById.start();
            }
        });

    }
}

File: m_wave/src/main/java/pdog18/com/module_wave/WaveActivity.kt
package pdog18.com.module_wave

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class WaveActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_wave)
        val viewById = findViewById<WaveView>(R.id.bezier)
        viewById.post(Runnable { viewById.start() })

    }
}
